Mlem/App/Views/Pages/ResponseComposerView.swift

```diff
@@ -14,19 +14,31 @@ struct ResponseComposerView: View {
     @Environment(Palette.self) var palette
     @Environment(\.dismiss) var dismiss
     
+    enum ResolutionState: Equatable {
+        case success, notFound, error(ErrorDetails), resolving
+    }
+    
     let textView: UITextView = .init()
     
     @State var text: String = ""
     @FocusState var focused: Bool
-    var context: ResponseContext
     
+    let originalContext: ResponseContext
+    let expandedPostTracker: ExpandedPostTracker?
+    @State var resolvedContext: ResponseContext
+    @State var resolutionState: ResolutionState = .success
+    @State var sending: Bool = false
+
     @State var account: UserAccount
     @State var presentationSelection: PresentationDetent = .large
     
     init?(
-        context: ResponseContext
+        context: ResponseContext,
+        expandedPostTracker: ExpandedPostTracker? = nil
     ) {
-        self.context = context
+        self.originalContext = context
+        self.resolvedContext = context
+        self.expandedPostTracker = expandedPostTracker
```

Comment:
This line fails the build on Xcode 15

```suggestion
        self.originalContext = context
        self._resolvedContext = .init(wrappedValue: context)
        self.expandedPostTracker = expandedPostTracker
```